content/ja/docs/demo/forking.md

```diff
@@ -0,0 +1,25 @@
+---
+title: デモリポジトリをフォークする
+linkTitle: フォーク
+default_lang_commit: 99b15da7137cde4a6352576768f4c9203170d569
+---
+
+[demo repository][]は、OpenTelemetryを使用した実装例を紹介するためのツールとして、フォークして使用できるように設計されています。
```

Comment:
I found there is no link url here.
So I follow this sentences as it is.

https://github.com/open-telemetry/opentelemetry.io/blob/main/content/en/docs/demo/forking.md?plain=1#L6

Reply:
no worries. would you mind creating an issue that addresses this?

Reply:
I created the issue!
If you think you need more info, let me know!

https://github.com/open-telemetry/opentelemetry.io/issues/6004